content/blog/2019/10/2019-10-21-thinking-about-jenkins-security.adoc

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Thinking About Jenkins Security - DevOps World 2019"
+tags:
+- jenkins world
+- jenkinsworld
+- devopsworld2019
+- security
+- event
+- community
+authors:
+- markewaite
+- wadeck
+- stackscribe
+opengraph:
+  image: /images/security/do-not-run-malicious-code.png
+---
+
+Securing a Jenkins server is a multi-layered effort that needs to consider networks, operating systems, applications, source code, and more.
+We'll review and explore the layers that secure Jenkins in the link:https://sched.co/UXWy[Thinking about Jenkins Security] talk at link:https://www.cloudbees.com/devops-world/lisbon[DevOps World | Jenkins World 2019] in Lisbon, Portugal.
+
+Topics will include:
+
+* The **secure by default** configuration that Jenkins provides
+* Risks associated with reducing default security settings
+* Authentication and authorization alternatives
+* Using "**least privilege**" principles
+* Jenkins **credentials** and trusted access to resources
+* Software updates and Jenkins project security notices
+
+image::/images/security/do-not-run-malicious-code.png[]
+
+We'll intermix descriptions of good practices with some security horror stories.
+The horror stories will remind us of our mistakes, how to detect them, and how to prevent them.
+
+Come join us for the presentation in Lisbon!
+link:https://drive.google.com/file/d/1RisjNUfu-3_VOyTIvY0cdWWzM8HKDtY9/view?usp=sharing[Draft slides] are available for review.
```

Comment:
You could embed the slides to the blog so that there is some content

Reply:
Any pointer to a location that will show me how to embed Powerpoint slides in an ASCIIDoc file?  My google searches have not found an obvious answer.

Reply:
See 403812417f8f15d21a58ce754871047685ff1fb6 for my attempt to embed the slides into the post